bin/sh/install-boost.sh

```diff
@@ -10,7 +10,7 @@ set -e
 
 if [ ! -d "$BOOST_ROOT/lib" ]
 then
-  wget $BOOST_URL -O /tmp/boost.tar.gz
+  wget --no-check-certificate $BOOST_URL -O /tmp/boost.tar.gz
```

Comment:
Ah, sorry, I just checked if there are any other official looking sources around. Yes, cloning from source would be the best option then I guess.